<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maplibrary</class>
 <widget class="QMainWindow" name="maplibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>710</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>maplibrary</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="searchbox">
      <property name="maximumSize">
       <size>
        <width>914</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>search box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxtype">
      <property name="maximumSize">
       <size>
        <width>914</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listofthings">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnlogin_logout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>308</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnapplyforborrow">
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>308</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply for a borrow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnexit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>308</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnmanage">
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>308</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Manage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnsave">
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>308</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Manage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionBook"/>
    <addaction name="actionGroup"/>
    <addaction name="actionUser"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSave_Data"/>
    <addaction name="actionContact_Us"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuaccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRegister"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuaccount"/>
   <addaction name="Manage"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogin">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-user.svg</normaloff>:/new/svg/resource/svg/fi-rr-user.svg</iconset>
   </property>
   <property name="text">
    <string>login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-cross-circle.svg</normaloff>:/new/svg/resource/svg/fi-rr-cross-circle.svg</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionmanage">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/png/resource/png/books-stack-of-three.png</normaloff>:/new/png/resource/png/books-stack-of-three.png</iconset>
   </property>
   <property name="text">
    <string>Manage</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-cloud-check.svg</normaloff>:/new/svg/resource/svg/fi-rr-cloud-check.svg</iconset>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionContact_Us">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-signal-alt-2.svg</normaloff>:/new/svg/resource/svg/fi-rr-signal-alt-2.svg</iconset>
   </property>
   <property name="text">
    <string>Contact Us</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-cross-small.svg</normaloff>:/new/svg/resource/svg/fi-rr-cross-small.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-user.svg</normaloff>:/new/svg/resource/svg/fi-rr-user.svg</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-user-time.svg</normaloff>:/new/svg/resource/svg/fi-rr-user-time.svg</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-user-add.svg</normaloff>:/new/svg/resource/svg/fi-rr-user-add.svg</iconset>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-user-delete.svg</normaloff>:/new/svg/resource/svg/fi-rr-user-delete.svg</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-cloud-check.svg</normaloff>:/new/svg/resource/svg/fi-rr-cloud-check.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBook">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/png/resource/png/books-stack-of-three.png</normaloff>:/new/png/resource/png/books-stack-of-three.png</iconset>
   </property>
   <property name="text">
    <string>Book</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-apps.svg</normaloff>:/new/svg/resource/svg/fi-rr-apps.svg</iconset>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/svg/resource/svg/fi-rr-users.svg</normaloff>:/new/svg/resource/svg/fi-rr-users.svg</iconset>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
